test/dlc_tests/eks/mxnet/training/test_eks_mxnet_training.py

```diff
@@ -18,12 +18,18 @@ def test_eks_mxnet_single_node_training(mxnet_training):
 
     rand_int = random.randint(4001, 6000)
 
+    framework_version_search = re.search(r"\d+(\.\d+){2}", mxnet_training)
```

Comment:
you can actually use get_framework_and_version_from_tag from test_utils/__init__.py

also feel free to update the function with this regex logic

Reply:
Ah, yes, I forgot about this. Thanks for pointing it out. I'll see if this change can be made.